Tests/BackendIntegrationTests/BaseBackendIntegrationTests.swift

```diff
@@ -51,6 +51,12 @@ class BaseBackendIntegrationTests: XCTestCase {
         configurePurchases()
     }
 
+    override func tearDown() {
+        Purchases.clearSingleton()
```

Comment:
`Purchases.setDefaultInstance` does:
```swift
if isConfigured {
    Logger.info(Strings.configure.purchase_instance_already_set)
}
```
Which happens on every test. Now all integration tests clear the instance at the end.